prow/cluster/jobs/istio/istio/istio.istio.master.yaml

```diff
@@ -11,62 +18,34 @@ istio_container: &istio_container
       memory: "24Gi"
       cpu: "7000m"
 
-build_spec: &build_spec
-  containers:
-  - image: gcr.io/istio-testing/prowbazel:0.4.13
-    args:
-    - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
-    - "--clean"
-    - "--timeout=90"
-    # Docker in Docker needs privileged mode
-    securityContext:
-      privileged: true
-  nodeSelector:
-    testing: build-pool
-
-test_spec: &test_spec
-  containers:
-  - image: gcr.io/istio-testing/prowbazel:0.4.13
-    args:
-    - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
-    - "--clean"
-    - "--timeout=90"
-    # Docker in Docker needs privileged mode
-    securityContext:
-      privileged: true
-  nodeSelector:
-    testing: test-pool
-
-branch_spec: &branch_spec
-- "^master$"
-
 presubmits:
 
   istio/istio:
 
   - name: istio-unit-tests
-    agent: kubernetes
+    <<: *job_template
     context: prow/istio-unit-tests.sh
-    branches: *branch_spec
     always_run: true
     rerun_command: "/test istio-unit-tests"
     trigger: "(?m)^/(retest|test (all|istio-unit-tests))?,?(\\s+|$)"
     labels:
-      preset-service-account: "true"
       preset-istio-kubeconfig: "true"
     spec:
-      <<: *build_spec
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
```

Comment:
You shouldn't need to call entrypoint directly. The pod utils will automatically have the entrypoint binary call the command you specify with `command` and `args` in the container spec.

Reply:
I tried 
```yaml
command:
- prow/istio-unit-tests.sh
```
and 
```yaml
args:
- prow/istio-unit-tests.sh
```
and none of them work.

Reply:
meaning entrypoint is not being called

Reply:
Discussed in DMs: This is not the pod utility entrypoint, this is Istio's entrypoint tool so this is actually correct. (Pod utils entrypoint is in the `/tools` directory).